changelog/unreleased/other/6928.md

```diff
@@ -0,0 +1 @@
+- :warning: Knows issue: the sdk might crash when no locations are available. The fix will be in the upcoming releases.
```

Comment:
Maybe we'll just use Core SDK version that doesn't crash?..

Reply:
different versions might have breaking changes, so we must descend them between different SDKs. As far as I remember rc-alpha-beta version might have breaking changes in every iteration. @LukasPaczos please correct me if I'm wrong

Reply:
Or force an early upgrade to v23.3.1 stable which is already out. Looking at the changelog the versions should be compatible. Could you try this @RingerJK?

Reply:
Seriously, there might be breaking changes between rc.1 and stable?

Reply:
I can, but even if everything compiled, methods incompatibility might produce a crash in runtime

Reply:
>Seriously, there might be breaking changes between rc.1 and stable?

event between bug-fix versions 😄

Reply:
Let's ask NN to bump?

Reply:
it's Friday evening, not the best time for that 😞 

I make smoke runs with common `23.3.1`, it works for me. wdyt folks, should we release with that version? I'd say it's ok for `RC` @dzinad @LukasPaczos

Reply:
:shipit: